<?xml version="1.0" encoding="UTF-8"?> <ui version="4.0"> <class>healthtest</class> <widget class="QWidget" name="healthtest"> <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>800</width> <height>600</height> </rect> </property> <property name="minimumSize"> <size> <width>800</width> <height>600</height> </size> </property> <property name="styleSheet"> <string notr="true"> QWidget { background-color: #f5f9fc; font-family: 'Microsoft YaHei'; } QLabel { color: #333333; font-size: 14px; } QLineEdit { border: 1px solid #d1d5db; border-radius: 4px; padding: 8px; background-color: white; font-size: 14px; } QLineEdit:focus { border: 1px solid #4a90e2; } QPushButton { background-color: #4a90e2; color: white; border: none; border-radius: 4px; padding: 10px 20px; font-size: 14px; } QPushButton:hover { background-color: #3a7bc8; } QPushButton:pressed { background-color: #2c6bb6; } #header { background-color: #4a90e2; color: white; padding: 15px; border-radius: 0; font-size: 20px; font-weight: bold; } #formContainer { background-color: white; border-radius: 8px; padding: 20px; margin: 15px; box-shadow: 0 2px 4px rgba(0,0,0,0.1); } </string> </property> <property name="windowTitle"> <string>健康评估</string> </property> <widget class="QLabel" name="label_healthtest_title"> <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>800</width> <height>60</height> </rect> </property> <property name="styleSheet"> <string notr="true">#header</string> </property> <property name="text"> <string>健康评估</string> </property> <property name="alignment"> <set>Qt::AlignCenter</set> </property> </widget> <widget class="QWidget" name="formContainer" native="true"> <property name="geometry"> <rect> <x>50</x> <y>80</y> <width>700</width> <height>450</height> </rect> </property> <property name="styleSheet"> <string notr="true">#formContainer</string> </property> <layout class="QGridLayout" name="gridLayout"> <item row="0" column="0"> <widget class="QLabel" name="label_sex"> <property name="text"> <string>性别：</string> </property> </widget> </item> <item row="0" column="1"> <widget class="QLineEdit" name="sex_"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入男或女</string> </property> </widget> </item> <item row="0" column="2"> <widget class="QLabel" name="label_height"> <property name="text"> <string>身高(cm)：</string> </property> </widget> </item> <item row="0" column="3"> <widget class="QLineEdit" name="height"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入身高</string> </property> </widget> </item> <item row="1" column="0"> <widget class="QLabel" name="label_weight"> <property name="text"> <string>体重(kg)：</string> </property> </widget> </item> <item row="1" column="1"> <widget class="QLineEdit" name="weight"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入体重</string> </property> </widget> </item> <item row="1" column="2"> <widget class="QLabel" name="label_heart_rate"> <property name="text"> <string>心率(次/分钟)：</string> </property> </widget> </item> <item row="1" column="3"> <widget class="QLineEdit" name="heart_rate"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入心率</string> </property> </widget> </item> <item row="2" column="0"> <widget class="QLabel" name="label_temperature"> <property name="text"> <string>体温(℃)：</string> </property> </widget> </item> <item row="2" column="1"> <widget class="QLineEdit" name="temperature"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入体温</string> </property> </widget> </item> <item row="2" column="2"> <widget class="QLabel" name="label_vital_capacity"> <property name="text"> <string>肺活量(ml)：</string> </property> </widget> </item> <item row="2" column="3"> <widget class="QLineEdit" name="vital_capacity"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入肺活量</string> </property> </widget> </item> <item row="3" column="0" colspan="2"> <widget class="QLabel" name="label_pressure"> <property name="text"> <string>血压(mmHg)：</string> </property> </widget> </item> <item row="4" column="0"> <widget class="QLabel" name="label_pressure_height"> <property name="text"> <string>高压：</string> </property> </widget> </item> <item row="4" column="1"> <widget class="QLineEdit" name="pressure_high"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入高压值</string> </property> </widget> </item> <item row="5" column="0"> <widget class="QLabel" name="label_pressure_low"> <property name="text"> <string>低压：</string> </property> </widget> </item> <item row="5" column="1"> <widget class="QLineEdit" name="pressure_low"> <property name="maximumSize"> <size> <width>200</width> <height>16777215</height> </size> </property> <property name="placeholderText"> <string>请输入低压值</string> </property> </widget> </item> <item row="6" column="2" colspan="2"> <layout class="QHBoxLayout" name="horizontalLayout"> <item> <widget class="QPushButton" name="healthtest_sure"> <property name="text"> <string>开始评估</string> </property> </widget> </item> <item> <widget class="QPushButton" name="healthtest_return"> <property name="text"> <string>返回</string> </property> </widget> </item> </layout> </item> </layout> </widget> </widget> <resources/> <connections/> </ui>
